app/src/main/java/com/nextcloud/talk/chat/ChatActivity.kt

```diff
@@ -1613,8 +1615,9 @@ class ChatActivity :
         }
     }
 
-    private fun startPlayback(message: ChatMessage, doPlay: Boolean = true) {
-        if (!active) {
+    private fun startPlayback(message: ChatMessage, doPlay: Boolean = true,
+        backgroundPlayAllowed: Boolean = false) {
```

Comment:
ok so you want to allow to play in background for hasConsecutiveVoiceMessage ..?
I get the point it makes sense for users that really use voice messages a lot and will use the ConsecutiveVoiceMessage feature. 
So for me it's fine to have this check for now until we have https://github.com/nextcloud/talk-android/issues/3785

Reply:
yes, I am one of them 🤗